README.md

```diff
@@ -139,6 +139,8 @@ The wizard will ask you for:
 - `smtp address`: The hostname of your smtp server
 - `smtp port`: The port of your smtp server
 
+#### Configuring Sentry for use with Pulse
+Sentry can be used with Pulse to give role based access control to your logs. In order to use 
```

Comment:
Maybe this is just an artifact of the diff, but it looks like the sentence is cutoff.